src/manifest.ts

```diff
@@ -1208,7 +1211,8 @@ export class Manifest {
         const strategy = await buildStrategy({
           ...config,
           github: this.github,
-          path,
+          path:
+            existsSync(path) && lstatSync(path).isFile() ? dirname(path) : path,
```

Comment:
There is no cloned git repo so these `fs` commands are all going to return `false` all the time.